app/assets/stylesheets/psique/components/_button.scss

```diff
@@ -64,12 +49,112 @@
 
 .btn--a {
   @extend %btn;
+  @include linear-gradient(lighten($capota, 20), $capota);
+  border: 1px solid $capota;
+  &:hover{
+    @include linear-gradient(lighten($capota,10), darken($capota, 8));
+  }
+
+}
+
+.btn--b {
+  @extend %btn;
+  @include linear-gradient(lighten($vibrant-orange, 20), $vibrant-orange);
+  border: 1px solid $dark-orange;
+  &:hover{
+    @include linear-gradient(lighten($vibrant-orange,10), darken($vibrant-orange, 8));
+  }
+}
+
+.btn--c {
+  @extend %btn;
+  background-color: $light-grey;
+  border: 1px solid $mid-grey;
+  color: $mid-grey;
+  cursor: not-allowed;
+}
+
+// Secondary
+
+.btn--icon{
+  @extend %btn;
+  .fa::before{
+    padding-right:4px;
+    vertical-align: text-top;
+  }
+  span{
+    padding-left:3px;
+  }
+}
+
+// Social
+
+.btn--facebook{
+  @include linear-gradient(lighten($facebook, 8), $facebook);
+  .fa::before{
+    padding: 5px 12px 5px 0px;
+    border-right: 1px solid $navyblue;
```

Comment:
Facebook color has name $facebook (maybe need blue word {I mean like $blue-facebook}) $navyblue is a primary color, "azul marino" xD, but I try to approach the names to code side.